docs/CSBR.md

```diff
@@ -584,38 +909,90 @@ For the status of Code Signing Certificates:
 For the status of Timestamp Certificates:
   * If the Subordinate CA provides OCSP responses, the Subordinate CA SHALL update information provided via an OCSP response at least every twelve months and within 24 hours after revoking a Timestamp Certificate.
 
+If the OCSP responder receives a request for the status of a certificate serial number that is not "assigned", then the responder MUST NOT respond with a "good" status.

```

Comment:
I think we should switch the order of this sentence and the next. Defining the term "assigned" before using it would add clarity.

Reply:
Fixed in https://github.com/cabforum/code-signing/commit/716838f4ec690748205f9496da65789f75bb46d7